animation_nodes/nodes/falloff/mesh_falloff.pyx

```diff
@@ -0,0 +1,75 @@
+import bpy
+import cython
+from mathutils import Vector
+from ... math cimport Vector3
+from ... utils.bvh import isInsideVolume
+from ... base_types import AnimationNode
+from ... data_structures cimport BaseFalloff
+from . constant_falloff import ConstantFalloff
+
+class MeshFalloffNode(bpy.types.Node, AnimationNode):
+    bl_idname = "an_MeshFalloffNode"
+    bl_label = "Mesh Falloff"
+
+    def create(self):
+        self.newInput("BVHTree", "BVHTree", "bvhTree")
+        self.newInput("Float", "Size", "size")
+        self.newInput("Float", "Falloff Width", "falloffWidth", value = 1)
+        self.newInput("Boolean", "Use Surface", "useSurface", value = True)
+        self.newInput("Boolean", "Use Volume", "useVolume", value = False)
+
+        self.newOutput("Falloff", "Falloff", "falloff")
+
+    def execute(self, bvhTree, size, falloffWidth, useSurface, useVolume):
+        if bvhTree is None: return ConstantFalloff(0)
+        if not useSurface and not useVolume: return ConstantFalloff(0)
+        return MeshFalloff(bvhTree, 1e10, size, falloffWidth, useSurface, useVolume)
```

Comment:
What I meant is that you should pass this value directly to `self.bvhTree.find_nearest`, it shouldn't be part of the falloff class.

Reply:
Okay.